### DISCUSSION TITLE
allauth

### DISCUSSION THREAD

**Discussion by koopg** [kaedroho/django-bridge#54] [Q&A] [answered]
what is the correct approach for authentication when using django-birdge  ?
can we use allauth ?
if the above answer is yes, will the implementation be exact as in traditional django (no react)  ? 

can we use CBVs?

**Comment by koopg** [comment, marked as answer]
OK, so  after allot of testing, I can confirm ( and please correct me if I'm wrong)
but it looks like allauth working just like on any django 
and that is great news  for me.

so now my only question is: can we use CBV?

**Reply by kaedroho** [reply]
Hi @koopg Sorry for the delay in replying!

As you discovered, django-bridge should work well with all authentication methods that work with Django!

> so now my only question is: can we use CBV?

Yes, you can! This may have been missed out in the documentation, but there is a mixin class that you can add to class based views: https://github.com/django-bridge/django-bridge/blob/main/python/django_bridge/views.py#L6
